immudb/typeconv.py

```diff
@@ -30,6 +34,8 @@ def sqlvalue_to_py(sqlValue):
         return sqlValue.bs
     elif sqlValue.HasField("s"):
         return sqlValue.s
+    elif sqlValue.HasField("ts"):
+        return datetime.fromtimestamp(sqlValue.ts/1e6, timezone.utc)
```

Comment:
Is this division floating point? If no, then we're truncating the timestamp to second precision only.

Reply:
It's floating point, and the corresponding test case covers also fractions of seconds.